src/wazuh_db/schema_upgrade_v3.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
```

Comment:
Every new file has to contain a header including the Wazuh copyright among other things.